board/freescale/mx6ullevk/cpld_reg.h

```diff
@@ -123,46 +144,47 @@ typedef union
 		/* When asserted to '1' it selects a dummy clock to the WD
 		 * device on board. As a result, It cancels the WD operation.
 		 * (Software WD triggers no longer necessary, reset will not be
-		 * done to the board upon missing SW WD), -- This is not an
+		 * done to the board upon missing SW WD) -- This is not an
 		 * operational mode -- ONLY for debug */
 		u8 cfg_wd_src:1;
 
 		/* Strobe the WD reset device. When writing 1 to this register,
 		 * the CPLD issues a single 20us pulse to the WD device */
 		u8 cfg_wd_ignore:1;
 
-		u8 padding0:2;
+		u8 cfg_wd_allow_10_sec:1;   /* not documented bit */
+		u8 padding0:1;
 		u8 cfg_wd_wdi:1;
+
+		/* R/W: Set to 1 to assert RESET to switch PCB */
 		u8 cfg_fpga_8020_rst:1;
-		u8 from_fpga_8020_rst:1;	/* Not yet implemented */
+
+		/* R: (Not yet implemented) Signal coming from EH8020 B2B
+		 * (reserved for asserting reset from switch side) */
+		u8 from_fpga_8020_rst:1;
+
 		u8 padding1:1;
 	} s;
 } T_CPLD_LOGIC_WD_RW_REGS;
 
 
-typedef union
-{
-	u8 uint8;
-	struct
-	{
-		u8 cfg_rst_n:1;		// When asserted to '0' resets the Ethernity switch.
-	} s;
-} T_CPLD_LOGIC_ETHERNITY_REGS;
-
```

Comment:
This removes the definition added in the previous commit. I would be nicer to squash this change into the previous commit.

Reply:
Fixed, thanks.